internal/iolimits/iolimits.go

```diff
@@ -17,7 +17,7 @@ const (
 	// `ReadAtMost`.
 	megaByte = 1 << 20
 	// MaxManifestBodySize is the maximum allowed size of a manifest. The limit
-	// of 4 MB aligns with the one of a Docker registry:
+	// of 4 MB aligns with the one of a container registry:
```

Comment:
This is a reference to a specific implementation, not the generic concept.

Reply:
I am not sure what you means Docker distribution registry?  Docker.io?

Reply:
The line just below links to the implementation in question. (I don’t know what exactly runs `docker.io` but it’s attractive to guess there’s a relationship.)

Reply:
Removed change.  Not really worth changing some of these.  I wanted to change the ones that show up in users faces.  But just grepped the code for any "docker registry" string.